qp_deblur/deblur.py

```diff
@@ -373,6 +373,9 @@ def deblur(qclient, job_id, parameters, out_dir):
     prep_info = qclient.get(
         '/qiita_db/prep_template/%s/' % artifact_info['prep_information'][0])
     df = pd.read_csv(prep_info['prep-file'], sep='\t')
+    if prep_info['data_type'] not in {'16S', '18S', 'ITS'}:
+        error_msg = ('deblur was developed only for target gene data')
```

Comment:
we maybe also want to give some explanation? `... as it assumes almost all sequences map to the same genomic region.`

Reply:
Is this assumption actually in the code or somewhere? I don't know ...